### PR TITLE
[frontend] Add no data available placeholder text in empty simulations charts

### PR DESCRIPTION
**Pull request by isselparra**
### Proposed changes
- Add no data available placeholder text in empty scenario results chart

### Related issues

Close https://github.com/OpenBAS-Platform/openbas/issues/1371

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- ~~[ ] I wrote test cases for the relevant uses case~~
- ~~[ ] I added/update the relevant documentation (either on github or on notion)~~
- [x] Where necessary I refactored code to improve the overall quality

Before:
image

After:
image

### CODE REVIEW

**Comment by Dimfacion** [comment]
Seems to be working but I'm unsure that's what we want. 
@jborozco can you confirm that you want an empty block with just "No data to display" ?

**Comment by jborozco** [comment]
Hello viewed on Teams with Issel, We want to have the text but also a background to look nicer

**Comment by isselparra** [comment]
PR updated @jborozco @Dimfacion

**Comment by guillaumejparis** [comment]
Build failed https://drone.filigran.io/OpenBAS-Platform/openbas/7119/1/7 
I think you maybe just missed some intellij plugin for eslint

**Comment by isselparra** [comment]
@guillaumejparis the plugin was already installed, an intellij config needed to be updated

File: openbas-front/src/utils/Charts.js [hunk 1]
```diff
@@ -273,6 +275,8 @@ export const verticalBarsChartOptions = (
   isResult = false,
   isFakeData = false,
   max = undefined,
+  isChartAPlaceholder = false,
```
Comment (Dimfacion) [inline openbas-front/src/utils/Charts.js:278]: Can we maybe use the existing isFakeData property instead of adding a new one ?

**Comment by Dimfacion** [comment]
I sometimes have an issue where 'No data to display' appears in front of the graph even though there are datas. 
image
Disabling the cache seems to fix the issue so maybe a problem with the store ?

File: openbas-front/src/utils/Charts.js (hunk 1 above)
Comment (isselparra) [reply openbas-front/src/utils/Charts.js:278]: If we use `isFakeData` directly we are going to also impact the `Simulations` chart in the home dashboard.
The scope described in the issue is only to impact the `Simulations results` chart at the scenario dashboard level.

Should we also modify the `Simulations` chart ? cc @jborozco

**Comment by isselparra** [comment]
@Dimfacion I corrected the problem with the "No data to display" text appearing event when the data is present.

I added a loader and I don't see the issue anymore.

Let me know if it's OK on your side :pray:

File: openbas-front/src/utils/Charts.js (hunk 1 above)
Comment (isselparra) [reply openbas-front/src/utils/Charts.js]: Discussed with @jborozco, we are updating also the `Simulations` chart

**Review by guillaumejparis — APPROVED** [review]
LGTM

**Pull request merged**